beacon-chain/types/crystallized_state.go

```diff
@@ -205,6 +205,11 @@ func (c *CrystallizedState) Validators() []*pb.ValidatorRecord {
 	return c.data.Validators
 }
 
+// DepositsPenalizedInPeriod returns total deposits penalized in the given withdrawal period.
+func (c *CrystallizedState) DepositsPenalizedInPeriod() []uint32 {
```

Comment:
slashes may be a better term here

Reply:
Slashing in general refers to a reaction to a drastic/malicious action by the validator. Unless this refers to the deposits being slashed due to the conditions being violated, penalized would be the appropriate naming here